pom.xml

```diff
@@ -19,6 +19,7 @@
       <jenkins.version>2.282</jenkins.version>
       <java.level>8</java.level>
       <spotbugs.skip>true</spotbugs.skip> <!-- Clean up redundant null check warnings and re-enable after SECURITY-1339 is released -->
+      <powermock.version>2.0.9</powermock.version>
```

Comment:
Better to rewrite or delete PM-based tests.

Reply:
didn't want to waste my time on this repo 😄

Reply:
Not enough it seems:

```
java.lang.NoSuchMethodError: org.mockito.Answers.get()Lorg/mockito/stubbing/Answer;
	at org.powermock.api.extension.listener.AnnotationEnabler.standardInject(AnnotationEnabler.java:85)
	at org.powermock.api.extension.listener.AnnotationEnabler.beforeTestMethod(AnnotationEnabler.java:55)
	at org.powermock.tests.utils.impl.PowerMockTestNotifierImpl.notifyBeforeTestMethod(PowerMockTestNotifierImpl.java:82)
```

Reply:
😢 , can look tomorrow unless you can take a quick look

Reply:
I will try it.

Reply:
https://github.com/jenkinsci/plugin-pom/pull/442